### ISSUE TITLE
possible new step required

### ISSUE THREAD

**Issue by cmil6199** [Hari-Nagarajan/fairgame#712]
when the bot finds an item and attempts to add it to the cart, there is now a confirm add to cart required.

**Comment by DakkJaniels** [comment]
Cool, do you have some kind of screenshot or page source that shows what this is? Without those, this is very unhelpful.

**Comment by cmil6199** [comment]
I will have to try to get one the next time it happens. I missed out on getting a purchase this morning due to it not adding it to the cart.

**Comment by Talivaldis** [comment]
![screen](https://user-images.githubusercontent.com/15898167/115637771-fc0bc000-a2de-11eb-90bb-0017c4c7a8d8.jpg)
Is this enough? or did you need to see more of the page? url? xpath? im very new to web stuff. But I can try and get you what you need if you want.

**Comment by DakkJaniels** [comment]
It should not have a problem clicking that button. Did you change the browser zoom?

**Comment by Talivaldis** [comment]
No zoom. And my log makes no mention of trying to click that button. unless it happens in check_stock, run_asins, or navigate_pages. 

It seems like it just tries to go to the cart and look for the checkout button. But I'm not very familiar with this code/process.

Though, I have gotten it the cart and past the cart before. no success checking out. Just this seems to happen a lot.

**Comment by DakkJaniels** [comment]
Can you post the relevant portion of the log?

**Comment by Talivaldis** [comment]
2021-04-21 19:40:18,119|0.6.5|INFO|Item B08L8KC1J7 in stock and in reserve range: 749.99 + 0.00 shipping <= 760.0
2021-04-21 19:40:18,120|0.6.5|INFO|Adding to cart
2021-04-21 19:40:18,148|0.6.5|INFO|Attempting Add To Cart with offer ID...
2021-04-21 19:40:19,953|0.6.5|INFO|Clicking button
2021-04-21 19:40:19,993|0.6.5|INFO|Button clicked
2021-04-21 19:40:21,357|0.6.5|DEBUG|'check_stock' returned True
2021-04-21 19:40:21,357|0.6.5|DEBUG|'run_asins' returned 'B08L8KC1J7'
2021-04-21 19:40:21,357|0.6.5|DEBUG|Calling navigate_pages(<stores.amazon.Amazon object at 0x00000185A3FD7F10>, False)
2021-04-21 19:40:21,383|0.6.5|DEBUG|Navigating page title: 'Amazon.com Shopping Cart'
2021-04-21 19:40:21,383|0.6.5|DEBUG|Calling handle_cart(<stores.amazon.Amazon object at 0x00000185A3FD7F10>)
2021-04-21 19:40:21,383|0.6.5|INFO|Looking for Proceed To Checkout button...

That button clicked appeared to be from the previous window.
Here is me interupting it. so that I stay on this window for a screenshot.

2021-04-21 19:40:29,844|0.6.5|INFO|Adding to cart
2021-04-21 19:40:29,859|0.6.5|INFO|Attempting Add To Cart with offer ID...
2021-04-21 19:40:30,835|0.6.5|INFO|Clicking button
2021-04-21 19:40:30,885|0.6.5|INFO|Button clicked
2021-04-21 19:40:31,029|0.6.5|INFO|Caught the interrupt signal.  Exiting.

**Comment by cmil6199** [comment]
yeah mine stopped at that page and wouldn't continue to the cart. I think the button says 'add' but not 100%

it would just timeout and go to the next item.

**Comment by DakkJaniels** [comment]
The click for that button is happening at Attempting to Add to Cart with offer ID -> clicking button. This is not a new page and is expected behavior to bypass additional offers if you try and add at the product data page. Maybe it's just responding slowly or not at all? Would need to see page source to better understand.

**Comment by Talivaldis** [comment]
So it's not supposed to pull up the buying options and click the add button in the side bar then?

**Comment by DakkJaniels** [comment]
It does that to check the offers and get the offeringListingId for the product.

**Comment by GitbubJD** [comment]
Same issue, can confirm the bot is failing this check
